packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -314,6 +314,17 @@ class MemoryController extends DisposableController
 
   ValueListenable get groupingByNotifier => groupingBy;
 
+  final _searchNotifier = ValueNotifier<String>('');
+
+  /// Notify that the search has changed.
+  ValueListenable get searchNotifier => _searchNotifier;
+
+  set search(String value) {
+    _searchNotifier.value = value;
+  }
+
+  String get search => _searchNotifier.value;
```

Comment:
nit: I think a good pattern is to only expose values that should be exposed through a ValueListenable through the value listenable. That way consumers know to listen for the value rather than relying on the current value. So leave the searchNotifier but remove this getter replacing its uses with searchNotifier.value